app/controllers/course/assessment/submission/submissions_controller.rb

```diff
@@ -115,6 +117,17 @@ def download_all
     end
   end
 
+  def download_statistics
+    authorize!(:manage, @assessment)
+    submission_ids = @assessment.submissions.by_users(student_ids).pluck(:id)
```

Comment:
How do we want to handle this if `submission_ids` is `nil`?

Reply:
I don't think ```submission_ids``` can be null. It can only be empty array?

Reply:
Yup. Actually my concern is that if it is an empty array should we return some 'error message' (eg. `no submissions are selected`) instead of downloading an empty CSV file?

Reply:
There is no detail requirement for this. I will fix this. I will throw bad request, if the submission id is empty.

Reply:
Ok thanks! 👍